assessments/0005-fluxcd.md

```diff
@@ -0,0 +1,183 @@
+# Assessment template
+
+Prepared by: \Celeste Horgan ([@celestehorgan](https://github.com/cncf/techdocs))<br>
```

Comment:
You probably don't want the `\` before your name.
Also, you'll need to update the link to your GH id.